### PR TITLE
New package: autotiling

### PR DESCRIPTION
**Pull request by adriangoransson**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

I just started using it, but I haven't encountered any issues so far.

### CODE REVIEW

File: srcpkgs/autotiling/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'autotiling'
+pkgname=autotiling
+version=1.4
+revision=1
+build_style=python3-module
+makedepends="python3-pip"
```
Comment (FollieHiyuki) [inline srcpkgs/autotiling/template:6]: Don't use `pip` in template

**Review by FollieHiyuki — CHANGES_REQUESTED** [review]

File: srcpkgs/autotiling/template (hunk 1 above)
Comment (FollieHiyuki) [reply srcpkgs/autotiling/template:6]: `hostmakedepends="python3-setuptools"` instead

Comment (adriangoransson) [reply srcpkgs/autotiling/template:6]: Thanks for the super quick check! :)
I was wondering why it was fine to build locally, though are you sure setuptools is required? I'm under the impression from the manual that it's implied by the build style.

Comment (FollieHiyuki) [reply srcpkgs/autotiling/template]: `python3-wheel` is redundant.
You can add `checkdepends="python3-pytest"` to run test

Comment (adriangoransson) [reply srcpkgs/autotiling/template]: `python3-wheel` does seem to be necessary, as it's specified as a [setup dependency](https://github.com/nwg-piotr/autotiling/blob/master/setup.cfg). I'll try with your other suggestions. :)

**Comment by adriangoransson** [comment]
Oddly platform specific build problems. I'll get back to this later.

File: srcpkgs/autotiling/template (hunk 1 above)
Comment (FollieHiyuki) [reply srcpkgs/autotiling/template]: My bad on the suggestions :)
I believe `python3-setuptools` already has its `wheel`.
```
$ xbps-query -f python3-setuptools | grep wheel
/usr/lib/python3.9/site-packages/setuptools/wheel.py
```
I checked the repo again. There is no test. So you can skip the `do_check()` by adding `make_check=no` below the checksum

Comment (adriangoransson) [reply srcpkgs/autotiling/template]: Thanks a bunch for your help! It seems to be working now.

File: srcpkgs/autotiling/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+build_style=python3-module
+hostmakedepends="python3-wheel"
+depends="python3-i3ipc"
+short_desc="Automatically switch horizontal/vertical split orientation in i3/sway"
+maintainer="Adrian Göransson <adriangoransson@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/nwg-piotr/autotiling"
+distfiles="https://github.com/nwg-piotr/${pkgname}/archive/v${version}.tar.gz"
+checksum=739653ffa8fc1bb5396e3c64cf35e25b343dd0391de5c8e71362c553a86e656f
+make_check=no
```
Comment (Chocimier) [inline srcpkgs/autotiling/template:14]: Leave comment on why is it disabled.

Comment (adriangoransson) [reply srcpkgs/autotiling/template:15]: Sorry about the delay!

**Pull request merged**